src/libraries/System.Text.Encodings.Web/pkg/System.Text.Encodings.Web.pkgproj

```diff
@@ -1,13 +1,9 @@
 ﻿<Project DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
   <ItemGroup>
-    <ProjectReference Include="..\ref\System.Text.Encodings.Web.csproj">
```

Comment:
Same deal here as WinHttpHandler, double check the nuspec diff.

Reply:
see comment above. I diffed all packages to be sure that the observable changes are expected.

Reply:
Yep, confirmed.  For some reason the runtime-only dependency detection isn't working for S.R.CS.Unsafe, which was added in this change: https://github.com/dotnet/runtime/commit/c569bc12223379bf894d7f68cb718b9b376e56eb cc @GrabYourPitchforks

Reply:
This isn't actionable feedback for this PR, but something that might want to be considered before we ship.  @Anipik here's a case were dependency validation (and a CSProj transition) could teach a user to mark a dependency as `PrivateAssets="Compile"`.